examples/webserver/fault-tolerance/src/test/java/io/helidon/reactive/webserver/examples/faulttolerance/MainTest.java

```diff
@@ -180,14 +179,14 @@ void testRetry() {
     @Test
     void testTimeout() {
         String response = client.get()
-                .path("/timeout/50")
+                .path("/timeout/10")
                 .request(String.class)
                 .await(1, TimeUnit.SECONDS);
 
-        assertThat(response, is("Slept for 50 ms"));
+        assertThat(response, is("Slept for 10 ms"));
 
         WebClientResponse clientResponse = client.get()
-                .path("/timeout/105")
+                .path("/timeout/1000")
```

Comment:
Test change to avoid intermittent failure where the timeout was not fast enough.